homeassistant/components/smappee_official/strings.json

```diff
@@ -1,17 +1,10 @@
 {
     "config": {
         "step": {
-            "pick_implementation": {
```

Comment:
The keys that were included in the scaffold template need to stay as they are used by the helper.